mindsdb/integrations/utilities/rag/rerankers/reranker_compressor.py

```diff
@@ -19,13 +19,14 @@
 
 class LLMReranker(BaseDocumentCompressor):
     filtering_threshold: float = 0.0  # Default threshold for filtering
+    provider: str = 'openai'
     model: str = DEFAULT_RERANKING_MODEL  # Model to use for reranking
     temperature: float = 0.0  # Temperature for the model
-    openai_api_key: Optional[str] = None
+    api_key: Optional[str] = None
```

Comment:
@tmichaeldb, I didn't find that this name is used as input to LLMReranker
but could miss something